### DISCUSSION TITLE
A minor avalanche of gratitude!

### DISCUSSION THREAD

**Discussion by mowglion** [noah-nuebling/mac-mouse-fix#562] [Other]
Thank you ever so much for creating this intrinsic bit of software that, like so much else, Apple ever so indolently omitted from macOS. 

Such a neat and nifty application at that, absolutely brilliant.

Wish you all the best!

**Comment by noah-nuebling** [comment]
Thank you very much for you nice words! I'm glad you like Mac Mouse Fix and I wish you alle the best as well! :)
